### ISSUE TITLE
Css dependencies fail

### ISSUE THREAD

**Issue by TrySound** [tivac/modular-css#297]
Tried to use values with watcher and got this fail.
```
yarn webpack v0.22.0
$ "D:\_host\sandbox\css-modules-sandbox\node_modules\.bin\webpack" --watch
Webpack is watching the files…
Hash: 7783882b7e271b828c75
Version: webpack 2.5.1
Time: 111ms
          Asset       Size  Chunks             Chunk Names
      bundle.js    3.29 kB       0  [emitted]  main
     bundle.css  103 bytes          [emitted]
bundle.css.json   97 bytes          [emitted]
   [0] ./src/style.css 91 bytes {0} [built]
   [1] ./src/index.js 66 bytes {0} [built]
D:\_host\sandbox\css-modules-sandbox\node_modules\dependency-graph\lib\dep_graph.js:183
      throw new Error('Node does not exist: ' + node);
      ^

Error: Node does not exist: src/style.css
    at Object.DepGraph.dependantsOf (D:\_host\sandbox\css-modules-sandbox\node_modules\dependency-graph\lib\dep_graph.js
:183:13)
    at files.map.filter.forEach (D:\_host\sandbox\css-modules-sandbox\node_modules\modular-css-core\processor.js:162:45)

    at Array.forEach (native)
    at Object.Processor.remove (D:\_host\sandbox\css-modules-sandbox\node_modules\modular-css-core\processor.js:158:14)
    at Compiler.compiler.plugin (D:\_host\sandbox\css-modules-sandbox\node_modules\modular-css-webpack\plugin.js:34:24)
    at Compiler.applyPlugins (D:\_host\sandbox\css-modules-sandbox\node_modules\tapable\lib\Tapable.js:61:14)
    at Compiler.newCompilation (D:\_host\sandbox\css-modules-sandbox\node_modules\webpack\lib\Compiler.js:452:7)
    at D:\_host\sandbox\css-modules-sandbox\node_modules\webpack\lib\Compiler.js:486:26
    at Compiler.applyPluginsAsyncSeries (D:\_host\sandbox\css-modules-sandbox\node_modules\tapable\lib\Tapable.js:131:46
)
    at Compiler.compile (D:\_host\sandbox\css-modules-sandbox\node_modules\webpack\lib\Compiler.js:481:7)
    at D:\_host\sandbox\css-modules-sandbox\node_modules\webpack\lib\Compiler.js:44:17
    at next (D:\_host\sandbox\css-modules-sandbox\node_modules\tapable\lib\Tapable.js:138:11)
    at Compiler.compiler.plugin (D:\_host\sandbox\css-modules-sandbox\node_modules\webpack\lib\CachePlugin.js:32:5)
    at Compiler.applyPluginsAsyncSeries (D:\_host\sandbox\css-modules-sandbox\node_modules\tapable\lib\Tapable.js:142:13
)
    at Watching._go (D:\_host\sandbox\css-modules-sandbox\node_modules\webpack\lib\Compiler.js:42:16)
    at Watching.invalidate (D:\_host\sandbox\css-modules-sandbox\node_modules\webpack\lib\Compiler.js:130:8)
    at Watching.<anonymous> (D:\_host\sandbox\css-modules-sandbox\node_modules\webpack\lib\Compiler.js:114:8)
    at EventEmitter.watcher.once (D:\_host\sandbox\css-modules-sandbox\node_modules\webpack\lib\node\NodeWatchFileSystem
.js:45:4)
    at EventEmitter.g (events.js:286:16)
    at emitTwo (events.js:106:13)
    at EventEmitter.emit (events.js:191:7)
    at EventEmitter._onTimeout (D:\_host\sandbox\css-modules-sandbox\node_modules\watchpack\lib\watchpack.js:142:7)
    at tryOnTimeout (timers.js:228:11)
    at Timer.listOnTimeout (timers.js:202:5)
error Command failed with exit code 1.
```

**Comment by TrySound** [comment]
Btw found some fragmentation in fileTimestamps
```js
{ 'D:\_host\sandbox\css-modules-sandbox\src\constants.css': 1494447596479,
  'D:\_host\sandbox\css-modules-sandbox\src\index.js': 1494447384067,
  'D:\_host\sandbox\css-modules-sandbox\src\style.css': 1494449039966,
  'src\constants.css': 1494447596479,
  'src\index.js': 1494447384067,
  'src\style.css': 1494449039966 }
```
Not criminal, but not sure it's safe. Something like this in loader would be better
```js
            deps.map(dep => path.join(path.dirname(this.resourcePath), dep))
                .forEach((dep) => this.addDependency(dep));
```

**Comment by tivac** [comment]
Sorry for all the webpack trouble, I appreciate your patience in explaining to me all the things I don't know. 😞 

I'll try to take a look at this later today.

**Comment by TrySound** [comment]
@tivac It's okay. All products is done in the same way :)

**Comment by tivac** [comment]
I think between #300 and #301 this is probably fixed? After landing #301 my local webpack tests are failing so trying to debug that.

**Comment by tivac** [comment]
Just kidding, tests working fine now. Re-open if it's still an issue!
